pkg/broker/filter/filter_handler.go

```diff
@@ -214,6 +218,10 @@ func (h *Handler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
 		return
 	}
 
+	// get audience of triggers subscriber from t.Status.SubscriberAudience somehow (new field)
+
+	// get triggers (?!?) .status.auth.serviceAccountName
```

Comment:
@creydr do you want to leave these comments in for future work?

Reply:
Ah. Should have not got commited...